packages/kbn-apm-synthtrace/src/scenarios/degraded_logs.ts

```diff
@@ -70,23 +86,28 @@ const scenario: Scenario<LogDocument> = async (runOptions) => {
       };
 
       const datasetSynth2Logs = (i: number, timestamp: number) => {
-        const index = Math.floor(Math.random() * 3);
+        const {
+          serviceName,
+          logMessage: { level, message },
+          cluster: { clusterId, clusterName },
+          cloudRegion,
+        } = constructLogsCommonData();
         const isMalformed = i % 60 === 0;
         return log
           .create({ isLogsDb })
           .dataset('synth.2')
-          .message(MESSAGE_LOG_LEVELS[index].message as string)
-          .logLevel(isMalformed ? MORE_THAN_1024_CHARS : MESSAGE_LOG_LEVELS[index].level) // "ignore_above": 1024 in mapping
-          .service(SERVICE_NAMES[index])
+          .message(message as string)
+          .logLevel(isMalformed ? MORE_THAN_1024_CHARS : level) // "ignore_above": 1024 in mapping
+          .service(serviceName)
           .defaults({
             'trace.id': generateShortId(),
             'agent.name': 'synth-agent',
-            'orchestrator.cluster.name': CLUSTER[index].clusterName,
-            'orchestrator.cluster.id': CLUSTER[index].clusterId,
+            'orchestrator.cluster.name': clusterName,
+            'orchestrator.cluster.id': clusterId,
             'orchestrator.resource.id': generateShortId(),
-            'cloud.provider': CLOUD_PROVIDERS[Math.floor(Math.random() * 3)],
-            'cloud.region': CLOUD_REGION[index],
-            'cloud.availability_zone': `${CLOUD_REGION[index]}a`,
+            'cloud.provider': getCloudProvider(),
```

Comment:
Why do we treat the cloud provider differently from the other values like cloud region?